web/html/src/manager/virtualization/guests/console/guests-console.js

```diff
@@ -32,11 +32,20 @@ type State = {
   hostId: string,
 };
 
+function getTokenExpiration(token: String): Date {
+  const jwsParts = token.split('.');
+  const claims = JSON.parse(atob(jwsParts[1]));
+  return new Date(claims["exp"] * 1000);
+}
+
 class GuestsConsole extends React.Component<Props, State> {
   client: ConsoleClientType;
   popupSubmit: Function;
   websocket: WebSocket;
   pageUnloading: boolean;
+  intervalId: ?number;
+  // This is intentionally not in state since we don't want this alone to trigger a rerender
+  isRefreshing = false;
```

Comment:
This variable seems to be unused, maybe some leftover?

Reply:
hum... may be I forgot the guards @Etheryte  mentioned in his comment